.github/labeler.yml

```diff
@@ -23,11 +23,27 @@ common:
   - src/common
   - src/global
   - src/log
+  - src/python-common/**
```

Comment:
i'd add this to pybind, cephadm, and other consumers of python-common.

Reply:
done